manager/assets/modext/widgets/resource/modx.panel.trash.js

```diff
@@ -0,0 +1,390 @@
+MODx.panel.Trash = function (config) {
+    config = config || {};
+    Ext.applyIf(config, {
+        id: 'modx-panel-trash'
+        , cls: 'container'
```

Comment:
Inconsistent code style in this section with the space between the comma and the property